crates/ruff_cli/src/diagnostics.rs

```diff
@@ -58,7 +62,7 @@ impl AddAssign for Diagnostics {
 }
 
 /// Returns either an indexed python jupyter notebook or a diagnostic (which is empty if we skip)
-fn load_jupyter_notebook(path: &Path) -> Result<(String, JupyterIndex), Diagnostics> {
+fn load_jupyter_notebook(path: &Path) -> Result<(String, JupyterIndex), Box<Diagnostics>> {
```

Comment:
Why did this need to be boxed?

Reply:
Clippy said the type was too large and suggested boxing it. It was a warning.